pylint/reporters/json_reporter.py

```diff
@@ -46,7 +48,10 @@ def display_messages(self, layout):
         print(json.dumps(self.messages, indent=4), file=self.out)
 
     def display_reports(self, layout):
-        """Don't do anything in this reporter."""
+        output = io.StringIO()
+        TextWriter().format(layout, output)
+        score = output.getvalue().split("\n")[1]
```

Comment:
There might be a better way, but understanding the way the VNode works and how one can finally get the message encapsulated in ten layers of linked list and visitor pattern does not seem worth it.